### PR TITLE
chore: fixed documentation

### PR DESCRIPTION
**Pull request by kikoso**
Fixed documentation broke in https://github.com/googlemaps/android-maps-compose/commit/450d842c38ca8ef9fb5938425408f57c3a793bb7

### CODE REVIEW

**Review by dkhawk — APPROVED** [review]

File: maps-compose/src/main/java/com/google/maps/android/compose/Marker.kt [hunk 1]
```diff
@@ -135,7 +135,7 @@ public fun rememberMarkerState(
  * A composable for a marker on the map.
  * @param state the [MarkerState] to be used to control or observe the marker
  * @param contentDescription the content description for accessibility purposes
- * @param state such as its position and info window
+ * state such as its position and info window
```
Comment (bubenheimer) [inline maps-compose/src/main/java/com/google/maps/android/compose/Marker.kt:138]: Still not fixed

File: maps-compose/src/main/java/com/google/maps/android/compose/Marker.kt [hunk 2]
```diff
@@ -607,7 +607,7 @@ public fun AdvancedMarker(
  *
  * @param state the [MarkerState] to be used to control or observe the marker
  * @param contentDescription the content description for accessibility purposes
- * @param state such as its position and info window
+ * state such as its position and info window
```
Comment (bubenheimer) [inline maps-compose/src/main/java/com/google/maps/android/compose/Marker.kt:610]: Still not fixed

File: maps-compose/src/main/java/com/google/maps/android/compose/Marker.kt [hunk 3]
```diff
@@ -534,7 +534,7 @@ private fun MarkerImpl(
...
```
Comment (bubenheimer) [inline maps-compose/src/main/java/com/google/maps/android/compose/Marker.kt:537]: Still not fixed

**Review by bubenheimer — COMMENTED** [review]
Some of the documentation regressions remain unfixed

**Pull request merged**